libs/types-common/src/Data/Json/Util.hs

```diff
@@ -34,6 +43,7 @@ infixr 5 #
 -- | A newtype wrapper for 'UTCTime' that formats timestamps in JSON with
 -- millisecond precision instead of the default picosecond precision.
 newtype UTCTimeMillis = UTCTimeMillis UTCTime
+  deriving (Eq, Show)
```

Comment:
Do we need this `Show` instance? If so, I think it'd be better to have a different `Show` instance that just shows `UTCTime` or even that does what we're doing in the `ToJSON` instance

Reply:
only used in tests, i think, but of course there is no guarantee it'll stay like this.  the default instance looks like this:

```
"UTCTimeMillis 2018-03-19 10:42:39.149485 UTC"
```

After 3d5c10df62657bd3b390bbdec492e31643a74327 it looks like this:

```
"UTCTimeMillis 2018-03-19T11:33:54.563Z"
```

good?